vendor.conf

```diff
@@ -21,7 +21,7 @@ google.golang.org/grpc 5b3c4e850e90a4cf6a20ebd46c8b32a0a3afcb9e https://github.c
 google.golang.org/genproto 09f6ed296fc66555a25fe4ce95173148778dfa85
 github.com/opencontainers/selinux 6ccd0b50d53ae771fe5259ff7a4039110777aa2d
 github.com/opencontainers/go-digest v1.0.0-rc0
-github.com/opencontainers/runtime-tools fdbc3d6d9507f699bbfd557dce0640c02b5f60e4
+github.com/opencontainers/runtime-tools master
```

Comment:
Floating with master makes me nervous.  runtime-tools cuts tagged releases; can we just use their most-recent v0.6.0?

Reply:
Agree. We should be on a tag or a commit.

Reply:
Can we make this change a separate PR?

Reply:
moved to #1676

Reply:
and cited a specific commit